spec/models/recipe_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Recipe, type: :model do
+  subject do
+    build(:recipe)
+  end
+
+  describe 'when saving' do
+    context 'with valid attributes' do
+      it 'saves successfully' do
+        expect(subject).to be_valid
+      end
+    end
+
+    context 'with invalid attributes' do
+      it 'fails when the name is not unique' do
+        recipe = create(:recipe, name: "same")
+        expect(Recipe.new(name: "same")).to_not be_valid
```

Comment:
recipe = create(:recipe, name: "subject.name)
expect(subject).to_not be_valid

Remember not to mix development syntax (Model.new) with test syntax (create(:model))